src/webgpu/capability_info.ts

```diff
@@ -6,60 +6,86 @@ export const kTextureFormatInfo: {
   // Packed 32-bit formats
-  'rgb10a2unorm':           { renderable:  true, color: true  },
-  'rg11b10float':           { renderable: false, color: true  },
+  'rgb10a2unorm':           { renderable:  true, color:  true, depth: false, stencil: false, storage: false, bytes:  4 },
+  'rg11b10float':           { renderable: false, color:  true, depth: false, stencil: false, storage: false, bytes:  4 },
   // 64-bit formats
-  'rg32uint':               { renderable:  true, color: true  },
-  'rg32sint':               { renderable:  true, color: true  },
-  'rg32float':              { renderable:  true, color: true  },
-  'rgba16uint':             { renderable:  true, color: true  },
-  'rgba16sint':             { renderable:  true, color: true  },
-  'rgba16float':            { renderable:  true, color: true  },
+  'rg32uint':               { renderable:  true, color:  true, depth: false, stencil: false, storage:  true, bytes:  8 },
+  'rg32sint':               { renderable:  true, color:  true, depth: false, stencil: false, storage:  true, bytes:  8 },
+  'rg32float':              { renderable:  true, color:  true, depth: false, stencil: false, storage:  true, bytes:  8 },
+  'rgba16uint':             { renderable:  true, color:  true, depth: false, stencil: false, storage:  true, bytes:  8 },
+  'rgba16sint':             { renderable:  true, color:  true, depth: false, stencil: false, storage:  true, bytes:  8 },
+  'rgba16float':            { renderable:  true, color:  true, depth: false, stencil: false, storage:  true, bytes:  8 },
   // 128-bit formats
-  'rgba32uint':             { renderable:  true, color: true  },
-  'rgba32sint':             { renderable:  true, color: true  },
-  'rgba32float':            { renderable:  true, color: true  },
+  'rgba32uint':             { renderable:  true, color:  true, depth: false, stencil: false, storage:  true, bytes: 16 },
+  'rgba32sint':             { renderable:  true, color:  true, depth: false, stencil: false, storage:  true, bytes: 16 },
+  'rgba32float':            { renderable:  true, color:  true, depth: false, stencil: false, storage:  true, bytes: 16 },
   // Depth/stencil formats
-  'depth32float':           { renderable:  true, color: false },
-  'depth24plus':            { renderable:  true, color: false },
-  'depth24plus-stencil8':   { renderable:  true, color: false },
+  'depth32float':           { renderable:  true, color: false, depth:  true, stencil: false, storage: false, bytes:  4 },
+  'depth24plus':            { renderable:  true, color: false, depth:  true, stencil: false, storage: false, bytes:  4 },
+  'depth24plus-stencil8':   { renderable:  true, color: false, depth:  true, stencil:  true, storage: false, bytes:  4 },
 };
 export const kTextureFormats = Object.keys(kTextureFormatInfo) as GPUTextureFormat[];
 
+export function isPackedFormat(format: GPUTextureFormat): boolean {
+  return format === 'depth24plus' || format === 'depth24plus-stencil8';
+}
```

Comment:
Added `copyable: boolean`

Reply:
I'll think about niceifying it some other time. I also realized the types exported by this file are insufficiently const-y so that's going to take some fixing.